### PR TITLE
add more unittest for BGP attributes

### PR DESCRIPTION
**Pull request by xiaopeng163**

### CODE REVIEW

File: yabgp/message/attribute/community.py [hunk 1]
```diff
@@ -70,75 +59,20 @@ def parse(self, value):
                     data=value)
         return community
 
-    @staticmethod
```
Comment (thomas-mangin) [inline yabgp/message/attribute/community.py]: Hi, You should really avoid this if/elif/else and do something like:
def to_string (community_type):
   return WELL_KNOW_COMMUNITY_INT_2_STR.get(community_type,'UNKNOW')

File: yabgp/message/attribute/community.py [hunk 2]
```diff
@@ -70,75 +59,20 @@ def parse(self, value):
...
-            return 'NO_EXPORT_SUBCONFED'
-        elif community_type == 0xFFFFFF04:
-            return 'NOPEER'
-        # elif community_typeCode == 317000132:
-        #    return "TEST"
-        else:
-            return 'UNKNOW'
-
-    @staticmethod
-    def to_int(value):
```
Comment (thomas-mangin) [inline yabgp/message/attribute/community.py]: Same here.

File: yabgp/common/constants.py [hunk 3]
```diff
@@ -256,6 +256,32 @@
...
+    'ROUTE_FILTER_v4': 0xFFFF0003,
+    'ROUTE_FILTER_TRANSLATED_v6': 0xFFFF0004,
+    'ROUTE_FILTER_v6': 0xFFFF0005,
+    'NO_EXPORT': 0xFFFFFF01,
+    'NO_ADVERTISE': 0xFFFFFF02,
+    'NO_EXPORT_SUBCONFED': 0xFFFFFF03,
+    'NOPEER': 0xFFFFFF04
+}
+
+WELL_KNOW_COMMUNITY_INT_2_STR = {
```
Comment (thomas-mangin) [inline yabgp/common/constants.py]: WELL_KNOW_COMMUNITY_INT_2_STR =  = dict([(r,l) for (l,r) in WELL_KNOW_COMMUNITY_STR_2_INT.items()])

Comment (xiaopeng163) [inline yabgp/common/constants.py]: @thomas-mangin thanks a lot :bow:

Comment (thomas-mangin) [inline yabgp/common/constants.py]: To be complete, the STR look better so I would keep this one and generate the INT - My OCD kicking in :laughing:

Comment (xiaopeng163) [inline yabgp/common/constants.py]: yeah, thanks for your review  :+1:

**Pull request merged**